src/api-client/index.js

```diff
@@ -119,3 +119,20 @@ function extractError(response) {
         return new Error(errors[0]);
     }
 }
+
+function stringify(object) {
+    return JSON.stringify(
```

Comment:
This is nice, but it does not work for the nested properties (`formData.media` is nested property).